apps/src/gamelab/GameLabSprite.js

```diff
@@ -260,9 +260,10 @@ module.exports.createSprite = function (x, y, width, height) {
 
 /*
  * Copied code from p5play from Sprite() with targeted modifications that
- * use the additional state parameter
+ * use the additional state parameter and use modifyPosition to determine 
+ * whether to modify the sprite's velocity and position.
  */
-var AABBops = function (p5Inst, type, target, callback) {
+var AABBops = function (p5Inst, type, target, callback, modifyPosition=true) {
```

Comment:
By adding a `modifyPosition` param that's orthogonal to `type` we are implying that passing `false` for `modifyPosition` will work for any collision type.  Is this true?  Can we add tests to prove this?